src/dbnode/persist/fs/migration/options.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2020 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package migration
+
+type options struct {
+	toVersion1_1 bool
+	concurrency  int
+}
+
+// NewOptions creates new migration options
+func NewOptions() Options {
+	return &options{}
```

Comment:
This will set concurrency to 0 which seems like it should be invalid; maybe default to `concurrency: 1`?

Reply:
I'd actually opt to move `defaultMigrationConcurrency` to this package and make it exported `DefaultMigrationConcurrency` then reference it from the config package.

Reply:
I think in that case would it be better if it's moved here but not exported, then if it's unset in the config you don't call `.SetConcurrency()` on the opts?